app-engine/forms/app_user.py

```diff
@@ -0,0 +1,20 @@
+"""
+    Summer Forms
+"""
+import re
+
+from wtforms import Form, validators, StringField, HiddenField, SelectField
+
+NDB_CONSISTENCY_CHOICES = [
+    ('eventual', 'Eventual (Default)'),
+    ('memcache', 'Memcache-Assisted Pseudo-Strong'),
+    ('interstitial', 'Interstitial Pseudo-Strong')
+]
+
+class AppUserConsistencyForm(Form):
+    ip_address = StringField('IP Address')
+    browserprint = HiddenField('Hidden Browserprint')
+    current_browserprint = StringField('Current Browser Fingerprint')
+    stored_browserprint = StringField('Stored Browser Fingerprint')
```

Comment:
These fields aren't actually displayed any longer. But form stills function so I'll leave them. Should add comment.